driver-core/src/main/com/mongodb/MongoClientSettings.java

```diff
@@ -73,7 +73,8 @@
                     new Jsr310CodecProvider(),
                     new JsonObjectCodecProvider(),
                     new BsonCodecProvider(),
-                    new EnumCodecProvider()));
+                    new EnumCodecProvider(),
+                    new Jep395RecordCodecProvider()));
```

Comment:
This should be added to the `getDefaultCodecRegistry` Javadoc below.

Reply:
Done